railties/test/application/test_runner_test.rb

```diff
@@ -268,6 +279,7 @@ def test_generated_controller_works_with_rails_test
     def test_run_multiple_folders
       create_test_file :models, "account"
       create_test_file :controllers, "accounts_controller"
+      create_test_file :helpers, "foo_helper"
```

Comment:
I added this because the test below needed 3 folders so it'd fail without the fix. I figured it's a good idea to add a 3rd folder here too, to make sure that the code does pick up 2 folders, instead of dropping all the arguments.